src/generator68k/Makefile.am

```diff
@@ -93,74 +67,13 @@ CPU_OBJ    = cpu68k-0.o  cpu68k-1.o  cpu68k-2.o  cpu68k-3.o  \
 	     cpu68k-8.o  cpu68k-9.o  cpu68k-a.o  cpu68k-b.o  \
 	     cpu68k-c.o  cpu68k-d.o  cpu68k-e.o  cpu68k-f.o
 
-$(CPU_SOURCE) : gen68k$(EXEEXT)
+gen68k.stamp : gen68k$(EXEEXT)
 	$(AM_V_GEN)./gen68k$(EXEEXT) 2>&1 >/dev/null
+	touch gen68k.stamp
+
+# use a stamp file so gen68k isnt invoked repeatedly for every file
+$(CPU_SOURCE) : gen68k.stamp
 
 $(CPU_OBJ) : %.o : %.c
 	$(AM_V_CC)$(COMPILE) -c $< -o $@
 
```

Comment:
Could you make sure that the stamp files get deleted with make clean with something like:
```
CLEANFILES = gen68k.stamp def68k.stamp
```

Also those two files should be in .gitignore